resources/application-connector/README.md

```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
 ## Details
 
 The Application Connector Helm chart contains all the global components:
+- Applicaiton broker
+- Application operator
 - Application registry
+- Connection token handler
 - Connector service
 
-## Installation
+## Connector Service
 
-The Application Connector is a part of the Kyma core and it installs automatically.
+The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR). It supports two modes: standalone and central. In standalone mode, the Connector Service works on and for the cluster in which it is deployed. In the central mode, Connector Service supports multiple clusters. By default, the installation of the Connector Service installation is set to standalone.
+
+### Install Connector Service as central
+
+To install Connector Service configured in the central mode you must override values as presented in the following example:
```

Comment:
```suggestion
To install the Connector Service as a standalone component, you must override several values of the Kyma Installer. Run: 
```

Reply:
I would suggest slight merge of two versions:
```
To install the Connector Service configured in the central mode, you must override several values of the Kyma Installer. Run: 
```